### PR TITLE
Dev grpc vless support

### PR DESCRIPTION
**Pull request by Sinofine**
#1474

### CODE REVIEW

File: src/ui/widgets/widgets/StreamSettingsWidget.ui [hunk 1]
```diff
@@ -640,6 +640,32 @@
...
+             <property name="text">
+              <string>Mode</string>
+             </property>
+            </widget>
+           </item>
+           <item row="1" column="1">
+            <widget class="QComboBox" name="grpcModeCB">
+             <item>
+              <property name="text">
+               <string>gun</string>
```
Comment (DuckSoft) [inline src/ui/widgets/widgets/StreamSettingsWidget.ui]: notr

File: src/ui/widgets/widgets/StreamSettingsWidget.ui [hunk 2]
```diff
@@ -640,6 +640,32 @@
...
+              </property>
+             </item>
+             <item>
+              <property name="text">
+               <string>multi</string>
```
Comment (DuckSoft) [inline src/ui/widgets/widgets/StreamSettingsWidget.ui]: notr

File: src/ui/widgets/widgets/StreamSettingsWidget.ui [hunk 3]
```diff
@@ -640,6 +640,32 @@
...
+              </property>
+             </item>
+             <item>
+              <property name="text">
+               <string>guna</string>
```
Comment (DuckSoft) [inline src/ui/widgets/widgets/StreamSettingsWidget.ui]: notr

File: src/ui/widgets/widgets/StreamSettingsWidget.ui [hunk 4]
```diff
@@ -39,7 +39,7 @@
           </size>
          </property>
          <property name="currentIndex">
-          <number>0</number>
+          <number>6</number>
```
Comment (DuckSoft) [inline src/ui/widgets/widgets/StreamSettingsWidget.ui]: this line should be reverted (maybe).

File: src/ui/widgets/widgets/StreamSettingsWidget.ui [hunk 5]
```diff
@@ -628,7 +628,7 @@
            </item>
           </layout>
          </widget>
-         <widget class="QWidget" name="page">
+         <widget class="QWidget" name="grpcStackPage">
```
Comment (DuckSoft) [inline src/ui/widgets/widgets/StreamSettingsWidget.ui:631]: Does this line break thing?

**Comment by DuckSoft** [comment]
暂时先不要写 `guna` 吧，因为目前还没有核心有实现。

File: src/ui/widgets/widgets/StreamSettingsWidget.ui (hunk 5 above)
Comment (Sinofine) [reply src/ui/widgets/widgets/StreamSettingsWidget.ui:631]: no, there is no code depend that name.

Comment (DuckSoft) [reply src/ui/widgets/widgets/StreamSettingsWidget.ui:631]: okay

File: src/ui/widgets/widgets/StreamSettingsWidget.ui (hunk 4 above)
Comment (Sinofine) [reply src/ui/widgets/widgets/StreamSettingsWidget.ui]: thanks, reverted.

**Comment by Sinofine** [comment]
如果能合并希望以squash形式合并,这commits太污染history了

**Review by DuckSoft — APPROVED** [review]
LGTM

**Pull request merged**